docs/index.bs

```diff
@@ -1435,18 +1445,10 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 
       <dfn method for="FetchEvent"><code>respondWith(|r|)</code></dfn> method *must* run these steps:
 
+        1. Let |event| be the [=context object=].
         1. If the <a>dispatch flag</a> is unset, [=throw=] an "{{InvalidStateError}}" {{DOMException}}.
         1. If the [=FetchEvent/respond-with entered flag=] is set, [=throw=] an "{{InvalidStateError}}" {{DOMException}}.
```

Comment:
Should these be "event's dispatch flag..." and "event's respond-with entered flag" instead of "the flag"?

Reply:
Yeah, I got a bit lazy since I wasn't touching these lines, but I should fix this up while I'm here.